index.html

```diff
@@ -1,389 +1,462 @@
-                            together to share knowledge, explore the latest
-                            trends, and collaborate on exciting projects.
-                        </p>
-                        <p class="text-justify leading-8 mt-4">
-                            Our meetups cover a wide range of topics, from
-                            JavaScript fundamentals to advanced frameworks,
-                            front-end and back-end development, and everything
-                            in between. We believe in the power of networking
-                            and learning from fellow enthusiasts, making our
-                            meetups a great place to connect with like-minded
-                            individuals. JSLovers is proud to have received
-                            recognition from Meetup for our outstanding
-                            contributions to the tech community.
-                        </p>
-                        <p class="text-justify leading-8 mt-4">
-                            We're also grateful to our sponsors, including
-                            Google, Microsoft, and many more, who help make our
-                            events even more exciting and informative. Join us
-                            for our upcoming events, workshops, and discussions
-                            to expand your JavaScript skills and be part of a
-                            supportive community that shares your love for
-                            coding. Let's code, learn, and grow together with
-                            JSLovers!"
-                        </p>
-                        <h2 class="text-3xl py-4 mt-4 font-bold text-center">
-                            Watch Talks About JSLovers
-                        </h2>
+  <body>
+    <a id="top"></a>
+    <div class="fixed bottom-0 right-0 z-50">
+      <a
+        href="#top"
+        class="bg-[#194d9f] text-white rounded-full w-8 h-8 flex justify-center items-center"
+      >
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          class="h-4 w-4 text-white"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            stroke-width="2"
+            d="M5 11l7-7 7 7M5 19l7-7 7 7"
+          />
+        </svg>
+      </a>
+    </div>
+    <header class="bg-[#194d9f] py-5 relative w-full">
+      <div class="flex justify-between container items-center mx-auto px-8">
+        <h1 class="inline-block">
+          <a class="font-dm-serif-display font-bold text-white" href="/">
+            <img
+              src="./resources/assets/jsloversLogo.png"
+              alt="jslovers homepage"
+              width="100"
+            />
+            <span>JSLovers</span>
+          </a>
+        </h1>
+        <nav id="desktop-nav" class="hidden md:flex">
```

Comment:
Flyout Navigation should hide once the user has clicked on any link and navigated to the specific section